packages/blade/.storybook/react/main.js

```diff
@@ -64,6 +64,17 @@ module.exports = {
       }),
     ];
 
+    // config.module.rules[0].exclude = new RegExp('/node_modules/(?!(@stackblitz)).*/');
+
+    config.module.rules.push({
+      test: /@stackblitz\/sdk[\\/].*\.m?js$/,
+      loader: 'babel-loader',
+      options: {
+        presets: ['@babel/preset-env'],
+        plugins: ['@babel/plugin-proposal-optional-chaining'],
+      },
+    });
```

Comment:
They use some modern javascript which our storybook doesn't understand so need to be passed from babel ;__;